modules/reactor/src/meshgenerators/PolygonConcentricCircleMeshGeneratorBase.C

```diff
@@ -928,5 +944,136 @@ PolygonConcentricCircleMeshGeneratorBase::generate()
   if (flat_side_up)
     MeshTools::Modification::rotate(*mesh0, 180.0 / (Real)_num_sides, 0.0, 0.0);
   mesh0->set_isnt_prepared();
+
+  if (_has_rings && getParam<bool>("replace_inner_ring_with_delaunay_mesh"))
+  {
+    if (getParam<bool>("quad_center_elements"))
+      paramWarning("quad_center_elements",
```

Comment:
```suggestion
      paramError("quad_center_elements",
```

Reply:
If this is an error instead of a warning, the document also needs to be updated as it now says "ignored".

Reply:
dang. Updated in the last push.